include/xrpl/protocol/ErrorCodes.h

```diff
@@ -136,9 +136,8 @@ enum error_code_i {
     rpcINVALID_LGR_RANGE = 79,
     rpcEXPIRED_VALIDATOR_LIST = 80,
 
-    // Reporting
-    rpcFAILED_TO_FORWARD = 90,
-    rpcREPORTING_UNSUPPORTED = 91,
```

Comment:
This is used by Clio. 
We can either leave this code be or replace it in Clio with a Clio-native code instead.
The latter would slightly change the output of the 'subscribe' RPC which uses this error code.

Reply:
Looks like we are going to use our own error code on Clio's side. 👍

Reply:
I suggest to just keep these two in `ErrorCodes.h`, at least for the time being - since this is libxrpl include, there's a reasonable expectation that it defines the rippled API.

Also, these names belong to API version 2, even if unused. So removing them is an API breaking change. I'm happy to compromise to rename `rpcREPORTING_UNSUPPORTED` to a more generic name (keeping the error code 91).

Reply:
Renaming the `rpcREPORTING_UNSUPPORTED` is an option but i suppose we want to keep the associated error strings as they are to avoid breaking the API.. in that case the output of Clio will remain with 'reportingMode' in the output. We can live with that. Hopefully can be properly addressed with OpenAPI later.

Reply:
Ok. I'll restore these two values and add a comment.

Reply:
On further research, it appears that Clio uses only `rpcREPORTING_UNSUPPORTED`, so I'll restore only that one.